### PR TITLE
Add test suite

### PR DESCRIPTION
**Pull request by gonzalo-bulnes**
**When I** maintain the cross-language test suite 
**I want** the features syntax to be validated automatically 
**So that** no errors go unnoticed 

See https://github.com/apiaryio/dredd-hooks-template/pull/9#issuecomment-273720579

### CODE REVIEW

**Comment by gonzalo-bulnes** [comment]
For reference, here is [a failing build][fail], and [a passing build][pass].

  [fail]: https://travis-ci.org/gonzalo-bulnes/dredd-hooks-template/builds/193999105
  [pass]: https://travis-ci.org/gonzalo-bulnes/dredd-hooks-template/builds/193999139

File: features/execution_order.feature [hunk 1]
```diff
@@ -74,13 +74,13 @@ Feature: Execution order
       #}
 
       """
-    Given I set the environment variables to:
+    And I set the environment variables to:
```
Comment (gonzalo-bulnes) [inline features/execution_order.feature]: This change likely doesn't require updating the Dredd hooks repositories, it is only refactoring (and only has a minor impact on readability). I think the linting rule is good to have however, and this gave me the opportunity to check that the build was breaking as expected and the corresponding output was [readable](https://travis-ci.org/gonzalo-bulnes/dredd-hooks-template/builds/193999105#L155).

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "test suite",
+    "Aruba"
+  ],
+  "author": "Apiary",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/gonzalo-bulnes/dredd-hooks-template/issues"
+  },
+  "homepage": "https://github.com/gonzalo-bulnes/dredd-hooks-template#readme",
+  "private": true
```
Comment (gonzalo-bulnes) [inline package.json]: I made this package **private**, and followed the [drafter.js][eg] example for the author and license fields.

  [eg]: https://github.com/apiaryio/drafter.js

**Comment by honzajavorek** [comment]
This is really great. Thanks!

**Pull request merged**